components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -268,14 +269,26 @@ def _unwrapunits(self, obj, units=None):
             return obj
         return obj.getValue()
 
-    def _getQueryString(self):
+    def _getQueryString(self, opts=None):
```

Comment:
As this is going to be used more extensively now and its return value has changed to a tuple the function needs pretty thorough documentation and we should definitely mark it for detailed review in the changelog. We have many examples where this method is overridden and/or its return value is used.